syft/pkg/cataloger/alpm/package_test.go

```diff
@@ -13,16 +13,19 @@ import (
 func Test_PackageURL(t *testing.T) {
 	tests := []struct {
 		name     string
-		metadata pkg.AlpmMetadata
+		metadata *parsedData
 		distro   linux.Release
 		expected string
 	}{
 		{
 			name: "bad distro id",
-			metadata: pkg.AlpmMetadata{
-				Package:      "p",
-				Version:      "v",
-				Architecture: "a",
+			metadata: &parsedData{
```

Comment:
nit: it's easier for future developers to keep the struct field names enumerated out instead of implied